nix/e2e-tests/src/phases/cdc_lifecycle.py

```
@@ -0,0 +1,183 @@
+    digest, _ = c.served_nar_digest(c.parse_narinfo(ni))
+    check(digest == state["eager_digest"], "drain-mode NAR content identical (served from chunks)")
+
+    # Stop ncps before mutating chunks (force-reclaim is drain-only, exclusive).
+    deployment.stop()
+    rc, dry = deployment.run_subcommand("migrate-chunks-to-nar", ["--dry-run"])
+    check(rc == 0, f"migrate-chunks-to-nar --dry-run exited cleanly (rc={rc})")
+    m = re.search(r'total["\s:=]+(\d+)', dry, re.IGNORECASE)
+    candidates = int(m.group(1)) if m else 0
+    check(candidates > 0, f"dry-run reports drain candidates (total={candidates})")
+    check(_chunked_nar_count(db) > 0, "dry-run did not mutate; chunks still present")
+    rc, out = deployment.run_subcommand("migrate-chunks-to-nar", ["--force-reclaim"])
```

Ruff (0.15.15)

[warning] Unpacked variable `out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)